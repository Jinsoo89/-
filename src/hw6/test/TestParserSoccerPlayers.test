# Test the parser on a graph that contains several nodes.

LoadGraph players soccerPlayers.tsv

ListChildren players C.Ronaldo