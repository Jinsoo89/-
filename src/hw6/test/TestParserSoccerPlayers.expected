# Test the parser on a graph that contains several nodes.

loaded graph players

the children of C.Ronaldo in players are: D.Maria(M.United) D.Maria(R.Madrid) Isco(R.Madrid) L.Modric(R.Madrid) S.Ramos(R.Madrid) Z.Ibrahimovic(M.United)